app/not-found.tsx

```
@@ -0,0 +1,2397 @@
+          enemyColor, // Cor principal do inimigo do mundo
+          world.colors.enemy, // Cor do mundo (pode ser a mesma ou variar)
+          world.colors.coin, // Cor da moeda como variação
+        ] as const;
+        enemies = [
+          {
+            x: 80,
+            y: 360,
+            width: 30,
+            height: 30,
+            direction: 1,
+            type: world.theme,
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)